packages/table/src/table.tsx

```diff
@@ -1156,6 +1156,12 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         e.stopPropagation();
 
         const { focusedCell } = this.state;
+        if (focusedCell == null) {
```

Comment:
Try arrowing around in the `selectedRegionTransform` example.

![image](https://cloud.githubusercontent.com/assets/443450/25297005/03401e3a-26a0-11e7-82a8-65eb06fbb4e8.png)